pkgs/applications/misc/cubiomes-viewer/default.nix

```
@@ -0,0 +1,67 @@
+{
+  lib, stdenv
+, fetchFromGitHub
+, qmake, qtbase, wrapQtAppsHook
+, makeDesktopItem, copyDesktopItems
+}:
```

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, qtbase
, qmake
, wrapQtAppsHook
, copyDesktopItems
, makeDesktopItem
}:
```
nit: should each be on their own line, in order of usage except for lib which always goes first

---

For some reason, I thought of it as the following

```
{
  lib, stdenv # standard stuff
, fetchFromGitHub # fetchers
, qmake, qtbase, wrapQtAppsHook # qt stuff
, makeDesktopItem, copyDesktopItems # other stuff
}:
```

I tried looking at recent PRs for anything which had a similar structure, but found few which had it clustered with multiple in a single line.